.github/workflows/test.yml

```
@@ -69,3 +69,22 @@ jobs:
       run: |
         cd prusti-contracts-test/
         cargo build
+    - name: Check Prusti binaries on Ubuntu and MacOS
+      if: matrix.os == 'ubuntu-latest' || matrix.os == 'macos-latest'
+      run: |
+        # Should succeed
+        target/debug/prusti-rustc \
+          prusti-tests/tests/verify/pass/no-annotations/assert-true.rs
+        # Should fail
+        ! target/debug/prusti-rustc \
+          prusti-tests/tests/verify/fail/no-annotations/assert-false.rs
+    - name: Check Prusti binaries on Windows
+      if: matrix.os == 'windows-latest'
+      run: |
+        :: Should succeed
+        target\debug\prusti-rustc.exe \
+          prusti-tests\tests\verify\pass\no-annotations\assert-true.rs
+        :: Should fail
+        target\debug\prusti-rustc.exe \
+          prusti-tests\tests\verify\fail\no-annotations\assert-false.rs
+        if %errorlevel%==0 (exit /b 1)
```

Sorry for the additional confusion from my previous comment. We are actually running in PowerShell, as indicated by the CI failure. I don't have a Windows machine to test this quickly, but maybe something like:

```suggestion
        target\debug\prusti-rustc.exe \
          prusti-tests\tests\verify\fail\no-annotations\assert-false.rs
        ! $?
```

---

On second thought, it might be that PowerShell supports the exact same syntax as Bash here? 😅

---

> We are actually running in PowerShell, as indicated by the CI failure

Oh, and I'm copying these commands from a `.bat` script... Let's see if it works.